src/frontend-samples/app-ui-samples/toolbar-button-provider-sample/index.tsx

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------

```

Comment:
I didn't notice that. When I started, all the app files were named "index" so I followed suit. Since all the apps are to show how to do UI, should I name them App or UI?

Reply:
I settled on calling the setup file (e.g., the one that initializes AppUi and registers the uiProvider) App and the file with the import UI bits UI. Let me know what you think.